app/src/components/market/sections/market_create/steps/items/ask_question_step.tsx

```diff
@@ -152,26 +162,79 @@ const AskQuestionStep = (props: Props) => {
     categoriesCustom,
     category,
     loadedQuestionId,
+    lowerBound,
     outcomes,
     question,
     resolution,
+    startingPoint,
+    unit,
+    upperBound,
   } = values
 
   const history = useHistory()
-  const [isImport, setIsImport] = useState(!!loadedQuestionId)
+
+  const [currentFormState, setCurrentFormState] = useState(state ? state : loadedQuestionId ? 'IMPORT' : 'CATEGORICAL')
+
+  const FormState = {
+    categorical: 'CATEGORICAL',
+    import: 'IMPORT',
+    scalar: 'SCALAR',
+  }
 
   const totalProbabilities = outcomes.reduce((total, cur) => total + (cur.probability ? cur.probability : 0), 0)
   const totalProbabilitiesNotFull = Math.abs(totalProbabilities - 100) > 0.000001
   const outcomeNames = outcomes.map(outcome => outcome.name)
-  const isContinueButtonDisabled =
-    totalProbabilitiesNotFull ||
-    outcomes.length < 2 ||
-    !question ||
-    !resolution ||
-    resolution < new Date() ||
-    !category ||
-    outcomeNames.map(name => !name).reduce((e1, e2) => e1 || e2) ||
-    outcomeNames.map((name, index) => outcomeNames.indexOf(name) !== index).reduce((e1, e2) => e1 || e2)
+
+  const [isContinueButtonDisabled, setIsContinueButtonDisabled] = useState(true)
+
+  const isProperScale =
+    lowerBound && startingPoint && upperBound && lowerBound?.lte(startingPoint) && startingPoint?.lte(upperBound)
+
+  useEffect(() => {
+    if (currentFormState === FormState.categorical || currentFormState === FormState.import) {
+      setIsContinueButtonDisabled(
+        totalProbabilitiesNotFull ||
+          outcomes.length < 2 ||
+          !question ||
+          !resolution ||
+          resolution < new Date() ||
+          !category ||
+          outcomeNames.map(name => !name).reduce((e1, e2) => e1 || e2) ||
+          outcomeNames.map((name, index) => outcomeNames.indexOf(name) !== index).reduce((e1, e2) => e1 || e2),
+      )
+    } else if (currentFormState === FormState.scalar) {
+      setIsContinueButtonDisabled(
+        !question ||
+          !lowerBound ||
+          !upperBound ||
+          !startingPoint ||
+          !unit ||
+          !resolution ||
+          resolution < new Date() ||
+          !category ||
+          !isProperScale ||
+          lowerBound.gte(startingPoint) ||
+          startingPoint.gte(upperBound) ||
+          lowerBound.lt(Zero),
```

Comment:
It's necessary given the way it's implemented, since we can't have a negative uint value on the smart contract, but I do see some value in supporting negative values from the front end, so maybe it would be good to implement that further down the line.